pkg/cmd/github-post/main.go

```diff
@@ -491,24 +492,34 @@ func getFileLine(
 	subtests := strings.Split(testName, "/")
 	testName = subtests[0]
 	packageName = strings.TrimPrefix(packageName, "github.com/cockroachdb/cockroach/")
-	cmd := exec.Command(`/bin/bash`, `-c`,
-		fmt.Sprintf(`cd "$(git rev-parse --show-toplevel)" && git grep -n 'func %s(' '%s/*_test.go'`,
-			testName, packageName))
-	// This command returns output such as:
-	// ../ccl/storageccl/export_test.go:31:func TestExportCmd(t *testing.T) {
-	out, err := cmd.CombinedOutput()
-	if err != nil {
-		return "", "", errors.Wrapf(err, "couldn't find test %s in %s: %s\n",
-			testName, packageName, string(out))
-	}
-	re := regexp.MustCompile(`(.*):(.*):`)
-	// The first 2 :-delimited fields are the filename and line number.
-	matches := re.FindSubmatch(out)
-	if matches == nil {
-		return "", "", errors.Errorf("couldn't find filename/line number for test %s in %s: %s",
-			testName, packageName, string(out))
+	for {
```

Comment:
This could devolve into an infinite loop if `packageName` happens to simplify to `"/"`. I'd add a precondition of the form,

```
if !strings.Contains(packageName, "pkg") {
    return "", "", errors.Newf("could not find test %s", testName)
}
```